reqs/tests/import_reqs_tests.py

```diff
@@ -251,3 +258,39 @@ def test_varying_policy_headers(opfid_field, uri_field):
     assert policy.policy_number == 123
     assert policy.omb_policy_id == "some identifier"
     assert policy.uri == "http://example.com/a"
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize("keyword,expected", (
+    ("Commodity IT", ["Commodity IT"]),
+    ("Data Management/Standards", ["Data Management/Standards"]),
+    ("Data Management/Standards. Reporting", ["Data Management/Standards",
+                                              "Reporting"]),
+    ("Definition", ["Definitions"]),
+    ("Emergency Preparedness?", ["Emergency Preparedness"]),
+    ("Governance -  Organization ", ["Governance - Organization"]),
+    ("Governance -  Organization", ["Governance - Organization"]),
+    ("Governance - Organization", ["Governance - Organization"]),
+    ("Governance -Organization", ["Governance - Organization"]),
+    ("Governance- Organization", ["Governance - Organization"]),
+    ("Governance-Organization", ["Governance - Organization"]),
+    ("Record Management", ["Records Management"]),
+    ("data management/standards", ["Data Management/Standards"]),
+    ("investments", ["Investments"]),
+), ids=repr)
+def test_keyword_normalization(blank_csv_file, keyword, expected):
+    """
+    Test that we don't add keywords with different capitalization into the
+    data.
+    """
+    csv_str = "\n".join([
+        SAMPLE_CSV2[0],
+        SAMPLE_CSV2[1].replace("Software Development Lifecycle/Agile", keyword)
+    ])
+    blank_csv_file.write(csv_str)
+
+    call_command('import_reqs', str(blank_csv_file))
```

Comment:
No biggie, but do you think that we could avoid all of this setup (e.g. by just calling `keywords`)? The more setup, the more complicated the tests are to understand -- for example, I had to jump to the top of the file to figure out why "Software" was in this list.